atomicapp/nulecule/main.py

```diff
@@ -90,6 +90,16 @@ def __init__(self, app_spec, destination=None, answers_file=None):
             if os.path.isfile(os.path.join(self.app_path, ANSWERS_FILE)):
                 self.answers_file = os.path.join(self.app_path, ANSWERS_FILE)
 
+        # TODO: put this in a better place in the future.
```

Comment:
The problem is that we don't call the provider code until we know what provider we are using. In this case we actually set that information here.